build/cmake/toolkit.cmake

```diff
@@ -70,7 +70,7 @@ set(wxTOOLKIT_LIBRARIES)
 set(wxTOOLKIT_VERSION)
 
 if(UNIX AND NOT APPLE AND NOT WIN32)
-    find_package(X11 REQUIRED)
+    find_package(X11)
```

Comment:
I wonder if we could keep it required when X11 support is enabled at GTK level. The trouble is that I don't really know how to check for this other than by running a test GTK program. Does anybody know of a better/simpler way by chance?

Reply:
We have a `wxHAVE_GDK_WAYLAND` for Wayland in FindGTK3.cmake, see below.  Something similar could be done for X11 support.

https://github.com/wxWidgets/wxWidgets/blob/master/build/cmake/modules/FindGTK3.cmake#L48

Reply:
Thanks, this is exactly what I was looking for! I'll add a similar `wxHAVE_GDK_X11`.